api/serializers.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from rest_framework import serializers
+from users.models import User
+from users.validators import CorrectUsernameAndNotMe
+
+
+class UserSerializer(serializers.ModelSerializer, CorrectUsernameAndNotMe):
+    first_name = serializers.CharField(
+        required=True,
+        max_length=settings.MAX_LEN_NAME
+    )
+    last_name = serializers.CharField(
+        required=True,
+        max_length=settings.MAX_LEN_NAME
+    )
+    username = serializers.CharField()
+    email = serializers.EmailField(
+        required=True,
+        max_length=settings.MAX_EMAIL_NAME_LENGTH
+    )
+    phone = serializers.CharField()
+    work_experience = serializers.CharField()
+    city = serializers.CharField()
+    raiting = serializers.CharField()
```

Comment:
```suggestion
    raiting = serializers.SerializerMethodField(read_only=True)
```
Рейтинг я так понимаю - средний балл со всех отзывов.